codalab/common.py

```diff
@@ -190,6 +190,27 @@ class StorageURLScheme(Enum):
     GCS_STORAGE = "gs://"
 
 
+# A Dict mapping to
+storage_url_type_dict = {
+    StorageURLScheme.DISK_STORAGE: StorageType.DISK_STORAGE,
+    StorageURLScheme.AZURE_BLOB_STORAGE: StorageType.AZURE_BLOB_STORAGE,
+    StorageURLScheme.GCS_STORAGE: StorageType.GCS_STORAGE,
+}
+
+
+def storage_url_to_type(url):
+    """Find corresponding StorageType by StorageURLScheme.
```

Comment:
This comment is a bit cryptic - can you expand on it and relate to `url`, perhaps with an example?

Reply:
"Return the storage type (e.g., ...) corresponding to the storage URL scheme (e.g., ...) given by `url` (e.g., ...).

Reply:
Done! Thanks Percy, I will pay more attention to comments.